pubspec.lock

```diff
@@ -42,7 +42,7 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
```

Comment:
The pubspec.lock file should not be edited in this situation. Put the original file in the branch